src/pages/DataExport/DataExport.js

```diff
@@ -44,18 +45,20 @@ export const PAGE_DESCRIPTION = i18n.t(
 const PAGE_ICON = <DataIcon />
 
 const today = new Date()
+const threeMonthsBeforeToday = new Date(
```

Comment:
I think it works because the [specs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date) say: 

> If any parameter overflows its defined bounds, it "carries over". For example, if a monthIndex greater than 11 is passed in, those months will cause the year to increment; if a minutes greater than 59 is passed in, hours will increment accordingly, etc. Therefore, new Date(1990, 12, 1) will return January 1st, 1991; new Date(2020, 5, 19, 25, 65) will return 2:05 A.M. June 20th, 2020.

So, for example when you have February, `today.getMonth()` will be 1 and then then `today.getMonth()-3` will be -2, which will cause it to subtract 1 from the year and set the month to November.